### ISSUE TITLE
[BUG] Journal entry template and note title

### ISSUE THREAD

**Issue by DutchPete** [qownnotes/scripts#250]
#### Script name
Journal entry

#### Script authors
@pbek, @sanderboom, @kantrol, @nico2sh  

#### Expected behavior
1. a new note without text added automatically at the top
2. to show the correct name of the note

#### Actual behavior
1. On the 1<sup>st</sup> and 2<sup>nd</sup> lines respectively the real file name is added,underlined with === 
2. the word _Note:_ followed by a long aplhanumeric code that changes to the real name by moving to another note and back.

#### Steps to reproduce
Create a new note using the script.

#### Output from the debug section in the settings dialog
Not available.

#### Comment
I “worked” on it in that I used ChatGPT to sort it out, which it did.
I went to the script location on my computer, added the amendment, then checked if it works. It does. It also includes code to not block/interfere with the keyoboard shotcut I use, `Alt+J`, to activate the script.

I shall not get involved in pull requests and all that goes with it, but I can send you the new script if it is of any help to you.

**Comment by pbek** [comment]
Can you please post the output from the debug settings that you can copy when you get to the settings dialog in QOwnNotes and head over to the *Debug* section of it?

Have you set a "Title Format"?

This was something added by @nico2sh in https://github.com/qownnotes/scripts/commit/f6e6406ce72cf079693bbcebb0fe16528a21651d.

**Comment by DutchPete** [comment]
Here is the debug output:

### Script `Journal entry`

**id**: `2`
**path**: `/home/user1/.var/app/org.qownnotes.QOwnNotes/data/PBE/QOwnNotes/scripts/journal-entry/journal-entry.qml`
**variablesJson**: 
```
{
    "defaultFolder": "Daily-notes-2024",
    "defaultTags": "",
    "noteBodyTemplate": "# 1. GEOPOLITICS\n## 1.1. \n\n<br>\n\n# 2. OTHER\n## 2.1. ",
    "noteTitleFormat": "{YYYY-MM-DD}, wk{WW}"
}
```
**identifier**: `journal-entry`
**version**: `1.7.0`
**minAppVersion**: `20.4.16`

What bothered me is that the tab title showed that long alphanumeric code.
Here is the improved version:
[journal-entry.qml.tar.gz](https://github.com/user-attachments/files/18176150/journal-entry.qml.tar.gz)

**Comment by pbek** [comment]
Thank you.

> I shall not get involved in pull requests and all that goes with it, but I can send you the new script if it is of any help to you.

Do you fear working with git in the terminal?
Because you don't really need to do that. You just need to click on the edit button on the top right on https://github.com/qownnotes/scripts/blob/master/journal-entry/journal-entry.qml.

![image](https://github.com/user-attachments/assets/115621e1-cbe8-4e17-bd0c-f20c3686b39b)

This should create a fork and after you commit your changes you can create a pull requests. All in the GitHub UI...

And then increase the version number in https://github.com/qownnotes/scripts/blob/master/journal-entry/info.json.

**Comment by DutchPete** [comment]
I'll give it a try. You'll have to tell me which of the 2 numbers I need to increase, and what the new number should be:

![image](https://github.com/user-attachments/assets/2c87b5a7-6c7c-4dea-b6fa-1d9b441d83a1)

**Comment by DutchPete** [comment]
OK, I did it, created the PR. Please tell me how to proceed.

**Comment by pbek** [comment]
Thank you!

> I'll give it a try. You'll have to tell me which of the 2 numbers I need to increase, and what the new number should be:

If you only fixed something, it's `1.7.1`.

> OK, I did it, created the PR. Please tell me how to proceed.

I made some comments.

**Comment by pbek** [comment]
> You'll have to tell me which of the 2 numbers I need to increase

`version`, see https://github.com/qownnotes/scripts?tab=readme-ov-file#editing-an-existing-script-in-the-github-web-interface.
